pom.xml

```diff
@@ -151,18 +149,16 @@
             </plugin>
             <plugin>
                 <artifactId>maven-release-plugin</artifactId>
-                <version>2.5.3</version>
                 <dependencies>
                     <dependency>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-scm-plugin</artifactId>
-                        <version>1.8.1</version>
+                        <version>1.11.2</version>
```

Comment:
Add property for `${maven.scm.plugin.version}` since this version is used in 2 places.

Reply:
I don't know why we're including the `maven-scm-plugin` dependency here. The `maven-release-plugin` does not depend on the `maven-scm-plugin`. I'm thinking we take it out and deal with the consequences later.